x-pack/legacy/plugins/monitoring/public/components/elasticsearch/shard_allocation/_shard_allocation.scss

```diff
@@ -76,27 +81,9 @@ monitoring-shard-allocation {
 
   .shard {
```

Comment:
Should rename this selector to be less generic.

Reply:
@andreadelrio Based on [your comment](https://github.com/elastic/kibana/pull/51100#discussion_r351031086), can you add a SASS-TODO comment to this and `.index` about where these classes are applied.